src/ray/object_manager/object_manager.h

```diff
@@ -313,6 +313,10 @@ class ObjectManager : public ObjectManagerInterface,
 
   int64_t GetMemoryCapacity() const { return config_.object_store_memory; }
 
+  double GetUsedMemoryPercentage() const {
```

Comment:
I think that we already have something like this in the resource manager that you could reuse.

Reply:
@stephanie-wang I only see resource manager in gcs, which probably doesn't fit here. Is this the one you mentioned?

Reply:
I was thinking of the [ClusterResourceScheduler](https://github.com/ray-project/ray/blob/master/src/ray/raylet/scheduling/cluster_resource_scheduler.h), although I'm not totally sure it's there. @ericl should know since I think he added the memory availability reporting for the autoscaler.